src/History.ts

```diff
@@ -62,13 +62,27 @@ export type ResetHistoryEntryAscend = ResetHistoryEntryBase & {
     kind: 'ascend'
 }
 
+export type ResetHistoryEntrySingularity = ResetHistoryEntryBase & {
```

Comment:
use an interface

Reply:
how would i use the ResetHistoryEntryBase interface in an interface? All the other ones did it this way

Reply:
https://www.typescriptlang.org/docs/handbook/interfaces.html#extending-interfaces